src/gui/uithememanager.h

```diff
@@ -48,10 +48,11 @@ class UIThemeManager : public QObject
     QIcon getIcon(const QString &iconId) const;
     QIcon getIcon(const QString &iconId, const QString &fallback) const;
     QIcon getFlagIcon(const QString &countryIsoCode) const;
-    QString getIconPath(const QString &iconId) const;
+    QPixmap getPixmap(const QString &iconId, const QWidget *widget, int baseHeight) const;
```

Comment:
Does it make sense that a `getPixmap` operation having a `QWidget` as one of the parameters? or maybe a better function name is in need?

Reply:
>or maybe a more precise function name is in need?

getScaledPixmap?